tests/unit/drivers/test_matches2doc_rank_drivers.py

```diff
@@ -41,17 +41,15 @@ def create_document_to_score():
     # |- matches: (id: 3, parent_id: 1, score.value: 3),
     # |- matches: (id: 4, parent_id: 1, score.value: 4),
     # |- matches: (id: 5, parent_id: 1, score.value: 5),
-
-    doc = jina_pb2.DocumentProto()
+    doc = Document()
```

Comment:
the idea herr is to see if we can refactor the logic of the test itself to be less dependant on id? I am not sure it is possible though

Reply:
One problem is that  this test is so dependant on document logic that any document refactor involves a lot of effort to regfactor these tests. is there a way to avoid this?

Reply:
the option I can see is to maintain fixtures in a separate file so that we can reuse/modify it anywhere

Reply:
butis there so many shared code?

Reply:
there could be, if we "composite" small fixtures into bigger ones, a lot of code are actually shared

Reply:
okey that'd be great.